README.rst

```diff
@@ -2116,6 +2116,101 @@ Remember to change ``geo_views`` location in ``urls.py`` in point 11 for extendi
 
 For more information about django views, please refer to the `views section in the django documentation <https://docs.djangoproject.com/en/dev/topics/http/views/>`_.
 
+Registering / Unregistering Commands
```

Comment:
this section is all alone almost at the end of the file.. I don't think this will help users to find this info.

I think we should rename `How to execute commands on device` to `Sending commands to devices` and move this section as a subsection to that one.

Then we could rename this heading to `Registering Custom Commands`.

Reply:
`How to execute commands on device` is a tutorial and it is along with other tutorials. I won't change it but I will add a separate section `Sending commands to devices`. In this, I will link to the `How to execute commands on device` and add another tutorial `Registering new command`. I would keep this section as it will be helpful for developers that are extending OpenWISP.